File: front_ERP_Supermercado/src/interface/lote.ts
export interface Lote{  
    id_lote?: number,
    stock: number,
    stock_minimo: number,
    cod_repisa: string,
    nombre_estado: string,
    descripcion_producto: string,
    cod_almacen: string,
    costo_unitario: number,
    fecha_caducidad: string
}

File: back_API_ERP_Supermercado/src/main/java/com/example/back_API_ERP_Supermercado/Entidad/Lote.java
package com.example.back_API_ERP_Supermercado.Entidad;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@Table(name = "lote")
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class Lote {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id_lote;
    private Integer stock;
    private Integer stock_minimo;
    private Integer id_repisa;
    private Integer id_estado;
    private Integer id_producto;
    private Integer id_almacen;
    private Integer costo_unitario;
    private String fecha_caducidad;
}
